app/routes/questions.actions.tsx

```diff
@@ -149,20 +163,20 @@ export const Action = ({
     const response = await fetch('/questions/actions', {method: 'POST', body: searchParams})
 
     if (response.ok !== true) {
-      setActionTaken(!actionTaken)
+      // don't ask
+      setActionTaken(actionTaken)
```

Comment:
?!?

Reply:
@zarSou9 explain yourself! For the allseeing gaze of Peter has fallen upon you!

Reply:
I might have seen that problem [once or twice](https://stackoverflow.com/a/58877875/1176601)... I think the code is fine for that purpose (using `(v) => !v)` would be wrong if the handler is called multiple times), but the comment could improve:
```suggestion
      // if response is not ok, revert to original state as it was before this handler started
      setActionTaken(actionTaken)
```